app/models/employee.rb

```diff
@@ -20,4 +23,11 @@ def date_of_birth_not_younger_than_15_years
       errors.add(:date_of_birth, "employee must be at least 15 years old")
     end
   end
+
+  def department_employee_limit
+    if department.present? && department.employees.count >= 10
```

Comment:
1.No magic numbers. Use constants
2.No need for `department.present?` check. Employees cannot be created without a department.